proto/cosmos/gov/query.proto

```diff
@@ -4,34 +4,51 @@ package cosmos.gov;
 import "cosmos/query/pagination.proto";
 import "gogoproto/gogo.proto";
 import "cosmos/gov/gov.proto";
+import "google/api/annotations.proto";
 
 option go_package = "github.com/cosmos/cosmos-sdk/x/gov/types";
 
 // Query defines the gRPC querier service for gov module
 service Query {
   // Proposal queries proposal details based on ProposalID
-  rpc Proposal (QueryProposalRequest) returns (QueryProposalResponse) {}
+  rpc Proposal (QueryProposalRequest) returns (QueryProposalResponse) {
+    option (google.api.http).get = "/cosmos/gov/v1beta/proposal/{proposal_id}";
+  }
 
   // Proposals queries all proposals based on given status
-  rpc Proposals (QueryProposalsRequest) returns (QueryProposalsResponse) {}
+  rpc Proposals (QueryProposalsRequest) returns (QueryProposalsResponse) {
+    option (google.api.http).get = "/cosmos/gov/v1beta/proposals";
+  }
 
   // Vote queries Voted information based on proposalID, voterAddr
-  rpc Vote(QueryVoteRequest) returns (QueryVoteResponse) {}
+  rpc Vote(QueryVoteRequest) returns (QueryVoteResponse) {
+    option (google.api.http).get = "/cosmos/gov/v1beta/vote/{proposal_id}/{voter}";
+  }
 
   // Votes queries votes of a given proposal
-  rpc Votes (QueryVotesRequest) returns (QueryVotesResponse) {}
+  rpc Votes (QueryVotesRequest) returns (QueryVotesResponse) {
+    option (google.api.http).get = "/cosmos/gov/v1beta/votes/{proposal_id}/{pagination.offset}/{pagination.limit}/{pagination.count_total}";
```

Comment:
I am not sure if we can avoid pagination flags from these annotations. Looks confusing.
```suggestion
    option (google.api.http).get = "/cosmos/gov/v1beta/proposals/{proposal_id}/votes";
```

Reply:
My mistake, they shouldn't be in the route. will remove it.